eth_utils/applicators.py

```diff
@@ -60,15 +75,18 @@ def apply_formatters_to_sequence(formatters, sequence):
             yield formatter(item)
 
 
-def apply_formatter_if(condition, formatter, value):
+def apply_formatter_if(condition: Callable[..., Any],
```

Comment:
Maybe too strict but `bool` might be more appropriate than `Any` here, however, that would result in **requiring** that the conditional return an actual boolean as opposed to something that we evaluate as *truthy/falsy*.

Not something I think we should do now, just wanted to bring it up.